### ISSUE TITLE
Allow use options in inDB

### ISSUE THREAD

**Issue by josejachuf** [gi0baro/weppy#150]
Allow use inDB validator with subqueries, order by, etc.

More details in [1]

[1 ] https://groups.google.com/forum/#!topic/weppy-talk/6SdB0lKzQsk

**Comment by gi0baro** [comment]
@josejachuf This is actually implemented since ea58c54.

This is the usage (following your example from the group):

```python
class Doctor(Model):
    name = Field()
    rating = Field('int')
    has_many('patients')

class Patient(Model):
    name = Field()
    age = Field('int')
    belongs_to('doctor')

    validation = {
        'doctor': {
            'in': {
                'dbset': lambda db: db(db.Doctor.id > 5), 
                'orderby': lambda doctor: ~doctor.rating,
                'label_field': 'name'
            }
        }
    }            
```

Will leave this opened until is fully tested and documented.

**Comment by josejachuf** [comment]
Hello  @gi0baro 
excuse the delay.
Where I tried it and it works fine.
